app/app/leftSidebar/left.sidebar.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Title</title>
+    <link rel="shortcut icon" href="../../static/images/favicon.ico" type="image/x-icon">
+    <link href="https://fonts.googleapis.com/css?family=Yanone+Kaffeesatz" rel="stylesheet">
+    <link rel="stylesheet" href="../../static/vendor/css/app.css">
+    <script type="text/javascript" src="../../static/vendor/js/app.min.js"></script>
+
+    <!-- Custom styles and scripts for App -->
+    <link rel="stylesheet" href="../../../bower_components/font-awesome/css/font-awesome.min.css">
+    <script src="../../../bower_components/jquery-slimscroll/jquery.slimscroll.min.js"></script>
+    <script src="../../../bower_components/metisMenu/dist/metisMenu.min.js"></script>
+    <!--
+    <script src="left.sdidebar.js"></script>
+    <script src="schema.js"></script>
+    <script src="left.sidebar.controller.js"></script>
+    -->
+</head>
+
+<body ng-app="app" class="pace-done fixed-nav fixed-nav-basic fixed-sidebar">
+
+<div id="wrapper">
+    <nav class="navbar-default navbar-static-side" role="navigation" ng-controller="sidebarCtrl">
+
+        <div class="sidebar-collapse" style="overflow: hidden; width: auto; height: 100%;">
+            <ul class="nav metismenu" id="side-menu" style="display: block;">
+                <li class="nav-header">
+                    <div class="dropdown profile-element">
+                        <img alt="image" class="img-circle" src="../../static/vendor/images/rr-logo.svg" width="50" height="50">
+                        <a data-toggle="dropdown" class="dropdown-toggle" href="#" aria-haspopup="true" aria-expanded="false">
+                            <span class="clear">
+                                <strong>Ім’я Прізвище</strong>
+                                <span class="text-muted block">Посада<span class="caret"></span></span>
+                            </span>
+                        </a>
+                        <ul class="dropdown-menu">
+                            <li><a href="#">Налаштування</a></li>
```

Comment:
@zpawn Also, please, remove # sign from href attribute for now. We use href='#/example' as link to some route. For now this route does not exist. So i just redirected to / page.